.pre-commit-config.yaml

```diff
@@ -5,7 +5,7 @@ repos:
     hooks:
       - id: check-merge-conflict
       - id: check-yaml
-      - id: detect-aws-credentials
+      # - id: detect-aws-credentials
```

Comment:
Delete this line, don't just comment it out.

Reply:
done. I just pushed a new commit with this small change, and for future repos I'll do the same. I added this information to https://github.com/dod-iac/learn-terraform-aws-instance/blob/main/getting-started.md

Reply:
Let's move this Getting Started guide over to `deptofdefense/dds-infra` where it would be seen by more folks.